src/CMakeLists.txt

```
@@ -120,7 +157,23 @@ target_include_directories(qzxing
         zxing/win32/zxing
         zxing/zxing
         zxing/bigint
-    PUBLIC
-    $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}>
+  PUBLIC
+        $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}>
 )
 
+if(BUILD_SHARED)
+    include (GNUInstallDirs)
+
+    set(QZXING_INSTALL_TARGETS qzxing)
+
+    install (
+        TARGETS ${QZXING_INSTALL_TARGETS} EXPORT QZXingTargets
+        LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
+        RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
+        ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
+    )
+
+    install (
+        FILES ${PUBLIC_HEADERS} DESTINATION "${CMAKE_INSTALL_INCLUDEDIR}/qzxing"
```

The QMake build installs the headers into the equivalent of just `CMAKE_INSTALL_INCLUDEDIR`. Could we make the install locations match to avoid differences between QMake & CMake being used for the build?

```
/nix/store/vyqb9vi69yrkmg4j1rw5vsd7rhfq7f5c-qzxing-3.3.0
├── include
│   ├── QZXing_global.h
│   └── QZXing.h
└── lib
    ├── libQZXing.so -> libQZXing.so.3.3.0
    ├── libQZXing.so.3 -> libQZXing.so.3.3.0
    ├── libQZXing.so.3.3 -> libQZXing.so.3.3.0
    ├── libQZXing.so.3.3.0
    └── pkgconfig
        └── QZXing.pc
```

```suggestion
        FILES ${PUBLIC_HEADERS} DESTINATION "${CMAKE_INSTALL_INCLUDEDIR}"
```